test/runTest.m

```diff
@@ -1,6 +1,29 @@
+%==========================================================================
+%This function runs the testFeatures tests, which test specific features of
+%the viewer, and the testDataSets tests, which test all datasets available in the
+%file.
+%
+%Authors: Thomas Maullin, Camille Maumet.
+%==========================================================================
+
 function runTest()
+    
+    %Setup steps for the test.
     import matlab.unittest.TestSuite;
-    clear classes;
-    tests = matlab.unittest.TestSuite.fromFile(which('nidmExampleDataTest'));
+    
+    %Create the testDataSets tests.
+    createTest();
+    
+    %Clear all classes we can.
+    warning('off','MATLAB:objectStillExists');
+    warning('off','MATLAB:ClassInstanceExists');
```

Comment:
To avoid having to clear those warnings, I think you can just call:
```
clear all
```
at the top of this function (before any import).